EnglishFilter/sections/specific.txt

```diff
@@ -303,6 +303,11 @@ inporn.com##section[is-footer-banners]
 designcorral.com##.vi_widgets_ads
 webdevtrick.com##.td-footer-ad
 wololo.net##.entry-inner > blockquote > center
+blockonomi.com##.sponban
+blockonomi.com##li[id^="custom_html"] > .textwidget > center
```

Comment:
I'd use this `blockonomi.com##.sidebar li[id^="custom_html"] > div.textwidget > center`.